src/components/Shared/Header/ListHeader.tsx

```diff
@@ -185,6 +187,8 @@ export const ListHeader: React.FC<ListHeaderProps> = ({
             buttonGroup={buttonGroup}
             contactsView={contactsView}
             selectedIds={selectedIds}
+            page={page}
+            isExcludedAppealPage={isExcludedAppealPage}
```

Comment:
Are there any components yet where this is `true`? Or is that in a future PR? I'm just trying to understand exactly what this flag does.

Reply:
Yes, it comes in a future PR